src/features/counter/button.css

```diff
@@ -0,0 +1,8 @@
+@value colors: "../../colors.css";
```

Comment:
@import '../scss/theme';

the folder structure of the styles folder
![image](https://user-images.githubusercontent.com/33009215/98209756-2a236080-1f48-11eb-805d-d9635cf55c27.png)